### ISSUE TITLE
Z3 plugin: All platform binaries included

### ISSUE THREAD

**Issue by lwrage** [smaccm/smaccm#54]
Platform specific binaries should be in platform specific fragments. That way only one binary gets installed. See, for example, the SWT binaries.

**Comment by agacek** [comment]
Yes. I tried this originally and it became too messy. I remember having difficulty getting it to work. I imagine somebody more versed with Eclipse will be needed to fix this.
